src/editions/transitioning.md

```diff
@@ -57,20 +40,22 @@ This will ensure that you're enabling all of the relevant features. Note that
 during the time the preview is available, we may continue to add/enable new
 features with this flag!
 
-### Running rustfix
+## Fix edition compatibility warnings
 
-There are some lints that can help you prepare for the next edition, but
-they're not currently turned on by default. `rustfix` has your back though!
-To turn them on and have `rustfix` fix up your code, run this:
+Next up is to enable compiler warnings about code which is incompatible with the
+new 2018 edition. This is where the handy `cargo fix` tool comes into the
+picture. To enable the compatibility lints for your project you run:
 
 ```shell
-$ cargo +nightly fix --prepare-for 2018
+$ cargo +nightly fix --prepare-for 2018 --all-targets
```

Comment:
Can we get a fix for https://github.com/rust-lang-nursery/edition-guide/issues/19#issuecomment-400842451 somewhere around here as well?